submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,156 @@
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/calendar.png" alt="calendar">
+                            <span>Calendar</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/chat.png" alt="chat">
+                            <span>Chat</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/classroom.png" alt="classroom">
+                            <span>Classroom</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/collection.png" alt="collection">
+                            <span>Collection</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/contacts.png"alt="contacts">
+                            <span>Contacts</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/documents.png" alt="documents">
+                            <span>Documents</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#">
+                            <img class="visible-img" src="icons/files.png"alt="files">
+                            <span>Files</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mail.png"alt="mail">
+                            <span>Mail</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/maps.png"alt="maps">
+                            <span>Maps</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a class="menu-a" href="#"> 
+                            <img class="visible-img" src="icons/mars.png"alt="mars">
+                            <span>Mars</span>
+                        </a>
+                    </li>
+                    <li class="more-btn greatest">
+                        <label class="more-btn" for="more">More</label>
```

Comment:
This button should be hidden when the second part of the popup is open 
![image](https://user-images.githubusercontent.com/16196199/183743095-8abd5615-b19a-4e99-9fcc-2ec931ba1ef0.png)